static/js/components/search/LearnerResult_test.js

```diff
@@ -137,6 +143,21 @@ describe('LearnerResult', () => {
     });
   });
 
+  it('should pass down a function that can open a dialog to email a learner', () => {
+    helper.store.dispatch(setLearnerChipVisibility(USER_PROFILE_RESPONSE.username));
+    let openLearnerEmailComposerStub = helper.sandbox.stub();
+
+    let result = renderLearnerResult({openLearnerEmailComposer: openLearnerEmailComposerStub});
+    return helper.listenForActions([SET_LEARNER_CHIP_VISIBILITY], () => {
+      result.find(".learner-name").props().onMouseEnter();
+    }).then(() => {
+      // Execute the openLearnerEmailComposer function passed down to the LearnerChip
+      result.find("LearnerChip").props().openLearnerEmailComposer();
+      assert.isTrue(openLearnerEmailComposerStub.called);
+      assert.isTrue(openLearnerEmailComposerStub.calledWith(USER_PROFILE_RESPONSE));
```

Comment:
I recommend using `sinon.assert.called()` and `sinon.assert.calledWith()` instead of `assert.isTrue()` for these cases. You get a much clearer failure message if the assertion fails. http://sinonjs.org/releases/v1.17.7/assertions/